policyengine_us/variables/gov/states/id/tax/income/deductions/standard/id_standard_deductions.py

```diff
@@ -0,0 +1,48 @@
+from policyengine_us.model_api import *
+
+
+class id_standard_deductions(Variable):
```

Comment:
this will be where(id_additional_standard_deductions_eligible, id_additional_standard_deductions, p.[standard deduction amount parameter based on filing status]